script/script.js

```diff
@@ -0,0 +1,177 @@
+function displaySignUp() {
+	document.getElementById("signup").style.display="block";
+}
+
+function displayLogIn() {
+	document.getElementById("login").style.display="block";
+}
+
+function closeDialog() {
+	document.getElementById("signup").style.display="none";
+	document.getElementById("login").style.display="none";
```

Comment:
Instead of closing both dialogs, prefer having separate methods for closing each or make a utility function for closing the dialogs.